client/app/cloud/project/compute/infrastructure/list/cloud-project-compute-infrastructure-list.controller.js

```diff
@@ -41,16 +60,38 @@ class CloudProjectComputeInfrastructureListCtrl {
             this.infra = infra;
             this.table.items = _.map(this.infra.vrack.publicCloud.items, instance => {
                 _.set(instance, "volumes", _.get(this.volumes, instance.id, []));
+                instance.ipv4 = instance.getPublicIpv4();
+                instance.ipv6 = instance.getPublicIpv6();
+                instance.macroRegion = this.RegionService.getMacroRegion(instance.region);
+                instance.statusToTranslate = this.getStatusToTranslate(instance);
+                instance.flavorTranslated = this.$translate.instant(`cpci_vm_flavor_category_${instance.flavor.name}`);
+
+                //patch for some translations that have &#160; html entities
```

Comment:
@Netnix Do you know why you have &#160; in flavor translations?

Reply:
I have no idea.